bioblend/_tests/TestGalaxyDatasetCollections.py

```diff
@@ -118,6 +124,34 @@ def test_update_history_dataset_collection(self):
         show_response = self.gi.histories.show_dataset_collection(history_id, history_dataset_collection["id"])
         self.assertFalse(show_response["visible"])
 
+    def test_show_dataset_collection(self):
+        history_id = self.gi.histories.create_history(name="TestDatasetCollectionShow")["id"]
+        dataset_collection1 = self._create_pair_in_history(history_id)
+        dataset_collection2 = self.gi.dataset_collections.show_dataset_collection(dataset_collection1['id'])
+        self.assertEqual(dataset_collection1.keys(), dataset_collection2.keys())
+        for element1, element2 in zip(dataset_collection1['elements'], dataset_collection2['elements']):
+            self.assertEqual(element1['id'], element2['id'])
+            self.assertEqual(element1.keys(), element2.keys())
+            for key in element1['object'].keys():
+                self.assertIn(key, element2['object'].keys())
+
+    @test_util.skip_unless_galaxy('release_18.01')
+    def test_download_dataset_collection(self):
+        history_id = self.gi.histories.create_history(name="TestDatasetCollectionDownload")["id"]
+        dataset_collection = self._create_pair_in_history(history_id)
+        tempdir = tempfile.mkdtemp(prefix='bioblend_test_dataset_collection_download_')
+        file_path = os.path.join(tempdir, 'dataset_collection.tgz')
+        self.gi.dataset_collections.download_dataset_collection(dataset_collection['id'], file_path=file_path)
+        self.assertTrue(os.path.isfile(file_path))
+        self.assertIn(os.path.getsize(file_path), (22, 46))
```

Comment:
What if you wait for the collection before downloading? I imagine this must affect the download size.

When I try locally with a fully uploaded collection I get a size of 310 bytes.

Reply:
The same tests still fail, but now with 176 instead of 310. It might be a problem with the wait method.

Reply:
In `dev` the downloaded archive is actually a `zip` and not a `tar.gz` archive as stated here: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/history_contents.py#L283

Reply:
In galaxy >= 21.01 is a zip archive, in galaxy < 21.01 it is a tgz archive.

Reply:
That's good to know, could you add a note in the docstring please?